src/tests/system/tests/test_netgroups.py

```diff
@@ -241,3 +243,131 @@ def test_netgroup__lookup_nested_groups_with_host_and_domain_values_present(
     result = client.tools.getent.netgroup("nested_group")
     assert result is not None
     assert expected in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(bz=802207)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__fully_qualified_names(client: Client, provider: GenericProvider):
+    """
+    :title: SSSD hangs when use fully qualified names set to true
```

Comment:
This is a description of the problem, not how SSSD should behave. Give it a spin.